content/docs/setup/kubernetes/additional-setup/cni/index.md

```diff
@@ -27,63 +27,135 @@ setup phase, thereby removing the [`NET_ADMIN` capability requirement](/docs/set
+        The Kubernetes documentation highly recommends this for all Kubernetes installations
+        where `ServiceAccounts` are utilized.
 
-    1.  Refer to the [Hosted Kubernetes Usage](#hosted-kubernetes-usage) section for any non-default
+    1.  Determine the Kubernetes environment's CNI plugin `--cni-bin-dir` and `--cni-conf-dir` settings.
+        Refer to the [Hosted Kubernetes Usage](#hosted-kubernetes-usage) section for any non-default
         settings required.
 
-1.  Create a namespace for the Istio control-plane components:
+1. Check the [Requirements for Pods and Services](/docs/setup/kubernetes/additional-setup/requirements/).
+
+1. [Install a Helm client](https://github.com/helm/helm/blob/master/docs/install.md) with a version higher than 2.10.
+
+## Helm chart release repositories
+
+The commands in this guide use the Helm charts that are included in the Istio release image.
+If you want to use the Istio release Helm chart repository instead, adjust the commands accordingly and
+add the Istio release repository as follows:
+
+{{< text bash >}}
+$ helm repo add istio.io https://storage.googleapis.com/istio-release/releases/{{< istio_full_version >}}/charts/
+{{< /text >}}
+
+## Installation steps
+
+Change directory to the root of the release and then
+choose one of the following two **mutually exclusive** options:
+
+1. To deploy Istio without using Tiller, follow the instructions for [option 1](/docs/setup/kubernetes/additional-setup/cni/#option-1-install-with-helm-via-helm-template).
+1. To use [Helm's Tiller pod](https://helm.sh/) to manage your Istio release, follow the instructions for [option 2](/docs/setup/kubernetes/additional-setup/cni/#option-2-install-with-helm-and-tiller-via-helm-install).
+
+### Option 1: Install with Helm via `helm template`
+
+Choose this option if your cluster doesn't have [Tiller](https://github.com/kubernetes/helm/blob/master/docs/architecture.md#components)
+deployed and you don't want to install it.
+
+1. Create a namespace for the `istio-system` components:
 
     {{< text bash >}}
     $ kubectl create namespace istio-system
     {{< /text >}}
 
-1.  Expand the release's compressed Istio CNI chart.
+1. Render and apply the Istio CNI components.
+   
+   {{< text bash >}}
+   $ helm template install/kubernetes/helm/istio-cni --name=istio-cni --namespace=istio-system | kubectl apply -f -
+   {{< /text >}}
+
+1. Install all the Istio
+    [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
+    (CRDs) using `kubectl apply`, and wait a few seconds for the CRDs to be committed in the Kubernetes API-server:
 
     {{< text bash >}}
-    $ tar -xzf install/kubernetes/helm/charts/istio-cni-*.tgz -C install/kubernetes/helm/
+    $ helm template install/kubernetes/helm/istio-init --name istio-init --namespace istio-system | kubectl apply -f -
```

Comment:
All istio install related text should be removed.  Istio and Istio-cni are now two very distinct steps and we don't want to refer to Isito install here in case it becomes stale and doesn't match how the main Istio install steps evolve.   The only thing that should be preserved and highlighted AS MUCH AS POSSIBLE is that the istio install need to be done with  `--set istio_cni.enabled=true`

Reply:
There is still a lot in common that was missing, e.g. the proper installation of Tiller. We need to give at least a copy-and-pastable example of a command to install `istio` with `--set istio_cni.enabled=true`, and stop there. And that's exactly what I've done here. I've removed all other profiles, for example. I don't see what else I can remove.

Reply:
I don't agree the more istio install content we put here the quicker it becomes out of date.  This was originally written at a time when the Istio and Istio-cni installs where tightly coupled.   Now the only two intersections points are that the istio-cni helm charts are available as part of the istio CNI and the istio install needs to set the istio_cni.enabled flag.  We should refactor this to only touch on those two intersection points.  If you want to provide links to create the coupling that is ok because at least if the link becomes stale the intent would still be clear.

Reply:
@john-a-joyce -- for release-1.1 the doc was updated with the separation of the istio-cni v. istio.  A lot of the stuff @rlenglet is adding is putting in repeated content from the Istio getting started guide and I disagree with that.  The existing istio-cni doc installation section-- https://istio.io/docs/setup/kubernetes/additional-setup/cni/#installation--it just refers to the details for installing the istio-cni and then an example command for passing the helm setting for `istio_cni.enabled=true`.   

@sdake's confusion triggering the issue creation was 
1) the istio helm chart / test-and-release changes broke the procedure by removing the istio-cni tarfile from the istio release charts 
2) he thought he could copy paste everything in just this doc and get Istio installed with the CNI.  (maybe auto-pilot from other doc usage)  

For 1) that's been addressed by the PR for the release/ stuff.  For 2) we can improve the commentary about the procedure but the  procedure still is the same as release-1.1.